pkg/update/required_test.go

```diff
@@ -12,6 +12,29 @@ import (
 
 func Test_getRequiredAirgapUpdates(t *testing.T) {
```

Comment:
this test should be updated given my comment above about using the channel id from the app